modules/eco-node-health-check-operator-installation-cli.adoc

```diff
@@ -63,12 +63,15 @@ metadata:
     name: node-health-check-operator
     namespace: openshift-operators
```

Comment:
![image](https://user-images.githubusercontent.com/58551443/154985103-b1eb7e7f-5173-4206-b652-06e482e20062.png)
if you want to create NHC in your own namespace, you need to create namespace, OperatorGroup and Subscription. And use the name of your namespace in "namespace" field.
If you want to create NHC in "openshift-operators" namespace, you don't need to create this namespace since it is already exists in openshift and also you don't need to create OperatorGroup for "openshift-operators" namespace.

Reply:
Thank you, updated!